lib/foreman_remote_execution/engine.rb

```diff
@@ -107,6 +107,10 @@ class Engine < ::Rails::Engine
       end
       Bookmark.send(:include, ForemanRemoteExecution::BookmarkExtensions)
       HostsHelper.send(:include, ForemanRemoteExecution::HostsHelperExtensions)
+
+      # We need to explicitly force to load the Task model due to Rails loader
+      # having issues with resolving it to Rake::Task otherwise
+      require_dependency 'foreman_tasks/task'
```

Comment:
The problem is we need this to run code exactly in the to_prepare section, for the Rails reloading to work both in development and production http://apidock.com/rails/Rails/Configuration/to_prepare